langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -333,6 +466,16 @@ public isolated function elementChildren(xml x, string? nm = ()) returns xml<Ele
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml bookSet = xml `<book><title>Java</title></book><book><title>Ballerina</title></book>`;
+# 
+# xml booksWithAuthors = bookSet.'map(function (xml xmlContent) returns xml {
+#           xml author = xml `<author>Anjana</author>`;
+#           return xml `<book>${xmlContent.children() + author}</book>`;
+# });
+# 
```

Comment:
```suggestion
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/7b4dfa608b04064393e46211524556d9850dfddf